1_fetch/src/get_site_data.R

```diff
@@ -43,3 +42,9 @@ get_site_data <- function(sites_info, state, parameter) {
 
   return(site_data_final)
 }
+
+# for splitting
+get_state_inventory <- function(sites_info, state) {
+  site_info <- dplyr::filter(sites_info, state_cd == state)
```

Comment:
Just a tip for the future: with really tiny steps, you can actually just put the code directly in the task makefile and avoid having to write a new function. If the small code chunk is self-explanatory, that is a nice option (as it is in this case). If not, it is sometimes nice to write a function because you can design the function name to be more descriptive about what is happening.

For example, instead of writing this function, you could do this in your task makefile:
```r
tar_target(nwis_inventory,
           oldest_active_sites %>%
               filter(state_cd == state_abb)
),
```

Reply:
wow, awesome